### ISSUE TITLE
Possible to show Flow Values on the endpoints?

### ISSUE THREAD

**Issue by robrien6** [weiweicui/PowerBI-Flowmap#4]
Hi,

Is this possible?  eg to have an option to show the value of the flow at the origin or destination rather than just on tooltip hover?

Also is it possible to have a suffix as well as a prefix on the tooltip value? Eg so you could say '23 passengers' rather than 'Passengers:23'

**Comment by weiweicui** [comment]
yes, you can change the setting "Bubble => Label" to "By click", then you can pin the tooltip for a piechart.

the postfix option is also added to the new v1.2 version. please check :)
